### PR TITLE
fix: thorchain hash

### PR DESCRIPTION
**Pull request by mpwanyi256**
This PR fixes the thorchain hash address, removing `ox` from transaction hashes if they exist

### CODE REVIEW

File: packages/wallet-core/src/swaps/thorchain/ThorchainSwapProvider.ts [hunk 1]
```diff
@@ -188,8 +188,10 @@ class ThorchainSwapProvider extends SwapProvider {
 
   async _getTransaction(hash: string): Promise<ThorchainTransactionResponse | null> {
     try {
-      return await this.thorchainAPIErrorParser.wrapAsync(() => this._httpClient.nodeGet(`/thorchain/tx/${hash}`), {
-        txHash: hash,
+      const hasOx = hash.charAt(0) + hash.charAt(1) === '0x'
+      const transactionHash = hasOx ? hash.slice(2) : hash
+      return await this.thorchainAPIErrorParser.wrapAsync(() => this._httpClient.nodeGet(`/thorchain/tx/${transactionHash}`), {
+        txHash: transactionHash,
```
Comment (koderholic) [inline packages/wallet-core/src/swaps/thorchain/ThorchainSwapProvider.ts:194]: @mpwanyi256 Is there a reason you're taking out the '0x'?

Comment (koderholic) [inline packages/wallet-core/src/swaps/thorchain/ThorchainSwapProvider.ts:194]: @mpwanyi256 Is there a reason you're taking out the '0x'?

**Review by koderholic — CHANGES_REQUESTED** [review]

File: packages/wallet-core/src/swaps/thorchain/ThorchainSwapProvider.ts (hunk 1 above)
Comment (mpwanyi256) [reply packages/wallet-core/src/swaps/thorchain/ThorchainSwapProvider.ts:194]: @koderholic yes, apparently thorchain updated to remove 0x that's why we the api was failing

Comment (koderholic) [reply packages/wallet-core/src/swaps/thorchain/ThorchainSwapProvider.ts:194]: Oh! Got it!

**Review by koderholic — APPROVED** [review]
LGTM

**Pull request merged**